File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        chut = {}
        for i in range(len(nums)):
            comp = target - nums[i]
            if comp in chut:
                return [chut[comp], i]
            else:
                chut[nums[i]] = i

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.go
func twoSum(numbers []int, target int) []int {
	i := 0
	j := len(numbers) - 1
	for i < j {
		sum := numbers[i] + numbers[j]
		if sum == target {
			return []int{i+1,j+1}
		}
		if sum < target {
			i += 1
		}
		if sum > target {
			j -= 1
		}
	}
	return []int{}
}
